readwiseApi.ts

```
@@ -29,29 +41,13 @@ export interface Tag {
-  highlights: Highlight[];
-  tags: Tag[];
-}
-
-export interface Books {
-  [key: string]: Book;
+export interface Exports {
+  [key: string]: Export;
 }
 
 export interface Library {
   categories: Set<String>;
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)